### PR TITLE
Qa checklist updates

### PR DESCRIPTION
**Pull request by georgeweiler**
Updates based on: 

https://coda.io/d/Building-Keep_d-fmEgBNFVH/Functional-and-Visual-QA-2022-21-01_suBPC#_luA8E
https://coda.io/d/Building-Keep_d-fmEgBNFVH/T-Migration-Threshold-side_suqpx#_luWi-

### CODE REVIEW

File: src/components/Typography/index.tsx [hunk 1]
```diff
@@ -21,7 +21,8 @@ export const H1: FC<TextProps> = (props) => {
   return (
     <Text
       as="h1"
-      fontSize="6xl"
+      fontSize="60px"
+      lineHeight="64px"
```
Comment (r-czajkowski) [inline src/components/Typography/index.tsx:25]: I believe we should use relative units not fixed eg. `px`. Eg. the screen size changes, the text still has 60px but should be smaller. So we should probably use [the object syntax](https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax) or [the array syntax](https://chakra-ui.com/docs/features/responsive-styles#the-array-syntax) to provide responsive styles.

File: src/pages/Overview/Network/UpgradeBanner.tsx [hunk 2]
```diff
@@ -47,9 +49,15 @@ const UpgradeBanner: FC = () => {
...
+            <>
+              <Body2>{heroText1}</Body2>
+              <Body2>{heroText2}</Body2>
+            </>
           ) : (
-            <H4 maxW="500px">{heroText}</H4>
+            <Stack maxW="450px" justify="center">
+              <H4>{heroText1}</H4>
+              <H4 noOfLines={2}>{heroText2}</H4>
+            </Stack>
```
Comment (r-czajkowski) [inline src/pages/Overview/Network/UpgradeBanner.tsx:60]: I think if we set the responsive styles correctly (please see comment above) we will avoid conditional rendering for mobile.

File: src/components/TokenBalance.tsx [hunk 3]
```diff
@@ -43,13 +45,27 @@ const TokenBalance: FC<TokenBalanceProps & TextProps> = ({
...
+            {shouldRenderTokenAmount ? _tokenAmount : "--"}
+          </H5>
         )}
+        {withSymbol &&
+          tokenSymbol &&
+          (isLarge ? (
+            <Body1 alignSelf="center">{tokenSymbol}</Body1>
+          ) : (
+            <Body3 alignSelf="center">{tokenSymbol}</Body3>
+          ))}
```
Comment (r-czajkowski) [inline src/components/TokenBalance.tsx]: Same as above.

File: src/components/Typography/index.tsx (hunk 1 above)
Comment (georgeweiler) [reply src/components/Typography/index.tsx:25]: I agree but we don't have designs for how each typography element should adjust at different breakpoints. I think we should ticket this out for later. We can have Liz and Sasha come up with the sizes they want and add the responsive values in a future PR. What do you think?

Comment (georgeweiler) [reply src/components/Typography/index.tsx:25]: started a discussion here: https://discord.com/channels/893441201628405760/935221615455789076/935221617498411079

File: src/pages/Overview/Network/WalletBalances.tsx [hunk 4]
```diff
@@ -110,28 +113,38 @@ const WalletBalances: FC = () => {
...
+
+      {/* Link to upgrade page */}
+      <Button size="lg" isFullWidth mt={4} as={RouterLink} to="/upgrade">
         Upgrade Tokens
       </Button>
+
+      {/* exchange rate link */}
       <HStack justify="center" mt={4}>
         <ExternalLink
+          as="p"
```
[the remark this answers is not in the document]
Comment (georgeweiler) [reply src/pages/Overview/Network/WalletBalances.tsx:139]: Good point I think this is just a copy/paste issue. I've updated.

**Review by Battenfield — APPROVED** [review]

**Pull request merged**